test/test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const ecpect = require('chai');
+const readFile = require('./lib/readWrite.js');
+
+* You will need to include tests that guarantee that the files are logged in the proper order.
+
+describe('readFile', function(){
+ it('should log files in the proper order', function(){
+  expect(readWrite).to.eventually.to.equal(true).and.notify();
```

Comment:
I am not exactly sure what the notify is doing here. I don't see it in the chai docs.